libs/explore/src/lib/explore-map-view.component.ts

```diff
@@ -40,6 +43,12 @@ const EMPTY = [];
             [actions]="actions | async"
             [labels]="labels | async"
         ></i-map>
+
+        <indoor-maps
+            *ngIf="map_key_exists | async"
+            [styles]="styles | async"
+            [actions]="actions | async"
+        ></indoor-maps>
         <explore-zoom-controls
```

Comment:
Does the maps indoor logic have their own zoom controls?
Might be worth hiding ours in that case.

Reply:
Yes they have their own zoom controls. Updated to hide our app controls whenever their map is used.